manifests/server.pp

```diff
@@ -458,8 +136,8 @@
   Variant[Boolean, String] $logfile                                 = false,
   String $port                                                      = '1194',
   Optional[String] $portshare                                       = undef,
-  String $proto                                                     = 'tcp',
-  String $status_version                                            = '',
+  Enum['tcp', 'udp'] $proto                                         = 'tcp',
+  Enum[1,2,3] $status_version                                       = 1,
```

Comment:
Oups : https://puppet.com/docs/puppet/5.3/lang_data_abstract.html#enum

Reply:
Many tests are failed because of default value. It also must be a string and not an integer.